src/engine/ruleset/integrations/windows/decoders/windows-sysmon.yml

```diff
@@ -117,41 +281,41 @@ normalize:
     - file.pe.product: $windows.EventData.Product
 
   # DNS event
- - check: $event.code=="22"
+ - check: $event.code == '22'
    map:
     - network.protocol: dns
 
- - check: $windows.EventData.SignatureStatus==Valid
+ - check: $windows.EventData.SignatureStatus == Valid
    map:
     - file.code_signature.valid: True
 
- - check: $windows.EventData.Initiated==True
+ - check: $windows.EventData.Initiated == 'True'
```

Comment:
If the value is a string, shouldn't we remove the single quotes?

Reply:
ditto